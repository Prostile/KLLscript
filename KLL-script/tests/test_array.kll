arr data[5]; // Статический массив
int i;
int temp;

// Ввод элементов
i = 0;
while (i < 5) begin
    cin(temp);
    data[i] = temp;
    i = i + 1;
end;

// Вывод элементов в обратном порядке
i = 4;
while (i > -1) begin // Используем > -1 вместо >= 0, т.к. >= нет
    cout(data[i]);
    i = i - 1;
end;

// Пример изменения элемента
data[1] = data[0] + data[4];
cout(data[1]);