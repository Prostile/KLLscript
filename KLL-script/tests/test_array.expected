55
44
33
22
11
66